python/pyspark/sql/connect/plan.py

```diff
@@ -259,6 +272,8 @@ def __init__(
 
     def plan(self, session: "SparkConnectClient") -> proto.Relation:
         plan = proto.Relation()
```

Comment:
can you just add a private helper method?

```
def _create_relation(self) -> proto.Relation:
  rel = proto.Relation()
  rel.common.plan_id = self._plan_id
  return rel
```

and then use this everywhere? It makes it easier later as well to add the source info as well. In addition it reduces the risk that a new plan node will miss adding the ID.

Reply:
nice, will add a helper method